src/ontology/mondo-edit.obo

```diff
@@ -164194,8 +164194,8 @@ intersection_of: MONDO:0001347 ! facioscapulohumeral muscular dystrophy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 ! SMCHD1
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 {source="MONDO:mim2gene_medgen", source="OMIM:158901"} ! SMCHD1
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:158901"} ! DUX4
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 {source="MONDO:mim2gene_medgen"} ! SMCHD1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
```

Comment:
same as above, these annotations should have been left alone.

Reply:
these should have been maintained are also in the mondo-omim-genes.robot.tsv file